### PR TITLE
[IMP][account_vocuher_tax] added tests to bank statement payment to taxes Task: #3006

### PR DESCRIPTION
**Pull request by JulioSerna**

### CODE REVIEW

**Comment by nhomar** [comment]
@JulioSerna 
Can you make a list of tests done here and/or a link to the related explanation of them, This PR is so important have it very very well documented for future migrations to new accounting.

**Comment by JulioSerna** [comment]
@nhomar 

In the name of branch has id of task #3006, there i have a document with all cases that i plan to tests

**Comment by nhomar** [comment]
@JulioSerna Can you add the list in some reference on the code one time it is finished please.

I just want to know if this PR is ready ar it is yet a WiP.

Good job dude.

**Comment by JulioSerna** [comment]
It is yet WiP

File: account_voucher_tax/tests/common.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+    def setUp(self):
+        super(TestTaxCommon, self).setUp()
+        self.account_invoice_model = self.registry('account.invoice')
+        self.account_invoice_line_model = self.registry('account.invoice.line')
+        self.acc_bank_stmt_model = self.registry('account.bank.statement')
+        self.acc_bank_stmt_line_model = self.registry(
+            'account.bank.statement.line')
+        self.partner_agrolait_id = self.registry(
+            "ir.model.data").get_object_reference(
```
Comment (josemoralesp) [inline account_voucher_tax/tests/common.py]: With the new api exists a method called ref, this method is used for get the record of the xml_id.
e.g.
self.env.ref('module.xml_id')

**Pull request merged**